.github/workflows/ci.yml

```diff
@@ -437,6 +464,7 @@ jobs:
   build-publish-master-image:
     needs:
       [
+        build-forc-test-project,
```

Comment:
Could you also add this to the `needs` arrays for the `notify-slack-on-failure` and `publish` jobs?

Reply:
@ra0x3 we're still missing this under the `publish` job.